ui/app/AppLayouts/Wallet/popups/simpleSend/SimpleSendModal.qml

```diff
@@ -1,17 +1,497 @@
+    - groupName: group name (from collection or community name)
+    - icon: from imageUrl or mediaUrl
+    - type: can be "community" or "other"
+    - subitems: submodel of collectibles/collections of the group
+    - key: key of collection (community type) or collectible (other type)
+    - name: name of the subitem (of collectible or collection)
+    - balance: balance of collection (in case of community collectibles)
+               or collectible (in case of ERC-1155)
+    - icon: icon of the subitem
+    **/
+    required property var collectiblesModel
+    /**
+    Expected model structure:
+    - chainId: network chain id
+    - chainName: name of network
+    - iconUrl: network icon url
+    Only networks valid as per mainnet/testnet selection
+    **/
+    required property var networksModel
+    required property var savedAddressesModel
+    required property var recentRecipientsModel
+    /** Input property holds currently selected Fiat currency **/
+    required property string currentCurrency
+    /** Input function to format currency amount to locale string **/
+    required property var fnFormatCurrencyAmount
+
+    /** input property to decide if send modal is interactive or prefilled **/
+    property bool interactive: true
+
+    /** input property to set estimated time **/
+    property string estimatedTime
+    /** input property to set estimated fees in fiat **/
+    property string estimatedFiatFees
+    /** input property to set estimated fees in crypto **/
+    property string estimatedCryptoFees
+
+    /** property to set and expose currently selected account **/
+    property string selectedAccountAddress
+    /** property to set and expose currently selected network **/
+    property int selectedChainId
+    /** property to set and expose currently selected token key **/
+    property string selectedTokenKey
+    /** property to set and expose the amount to send from outside without any localization **/
+    property string selectedAmount
+    /** output property to set currently set amount to send
+    Crypto value in a base unit as a string integer,
+    e.g. 1000000000000000000 for 1 ETH **/
+    readonly property string selectedAmountInBaseUnit: amountToSend.amount
+
+    /** TODO: replace with new and improved recipient selector StatusDateRangePicker
+    TBD under https://github.com/status-im/status-desktop/issues/16916 **/
+    property alias selectedRecipientAddress: recipientsPanel.selectedRecipientAddress
+    /** Input function to resolve Ens Name **/
+    required property var fnResolveENS
+    /** Output function to set resolved ens name values **/
+    function ensNameResolved(resolvedPubKey, resolvedAddress, uuid) {
+        recipientsPanel.ensNameResolved(resolvedPubKey, resolvedAddress, uuid)
+    }
+
+    /** Output signal to request signing of the transaction **/
+    signal reviewSendClicked()
+    /** Output signal to request fetching fees **/
+    signal fetchFees()
```

Comment:
@alexjba I agree with your point here completely. The Simple Send already is a kind of form, so we can defintely handle all of this logic in the SendModalHandler instead. 

I think it would be a good place to separate the UI part from the BL part. I'll remove it and put the logic in the SendModalHandler instead

Reply:
@alexjba Ive moved the logic out a but, however the check for if form was filled correctly I have kept it within the SimpleSendModal.qml because I think its an internal UI state for the form that can depend on various sub components etc?

Also instead of emitting fetchFees, Ive added formChanged signal instead